### ISSUE TITLE
Normdatenumfeld: Suche nach joint Titeln 

### ISSUE THREAD

**Issue by thefass** [ubtue/tuefind#1578]
Die Suche nach joint Titeln funktioniert nicht
https://ptah.ub.uni-tuebingen.de/Authority/081707754
Bei Klck auf Gebhard, Winfried 1961 müßte eigentlich dieser Titel gefunden werden:
https://www.ixtheo.de/Record/1587783738

**Comment by thefass** [comment]
**Links und Symbole**
- Das Buchsymbol ist sonst "Show more titles for this autor" und sollte nicht für "Show joint titles for this autor" verwendet werden (vielleicht besser so etwas wie ein Kettensymbol verwenden)
- Bitte noch als dritten Link ergänzen: "Show more titles for this autor" (Symbol: Buch)

**Comment by mtrojan-ub** [comment]
Wenn ich es auf ptah teste dann taucht der Titel korrekt auf. Die Links / Symbole werde ich noch anpassen.

**Comment by thefass** [comment]
Stimmt! ich war vermutlich durch die lange Schlurre davor abgelenkt
![Bildschirmfoto 2021-07-16 um 09 36 38](https://user-images.githubusercontent.com/14169098/125910507-51e99331-7408-40b7-b79b-5f001b395393.png)


Das müßte verschlankt werden

**Comment by mtrojan-ub** [comment]
Links + Symbole jetzt angepasst, siehe ptah. "Verschlankung" noch offen.

**Comment by thefass** [comment]
Links + Symbole passen. 

https://ptah.ub.uni-tuebingen.de/Authority/077468317
Wenn man dann dort auf "Show all related authors" klickt kommt man hierhin:
https://ptah.ub.uni-tuebingen.de/Author/Search?lookfor=Angenendt%2C+Arnold+1934-
Bitte auch noch auf der "Show all related authors" Seite die Links und Symbole einfügen!

**Comment by mtrojan-ub** [comment]
@LysogorAndGmail : The template seems to be `bootstrap3/templates/search/list-authorfacets.phtml`. Please copy it to the tuefind theme and make the changes there, do not change the original template.

**Comment by mtrojan-ub** [comment]
@thefass: Durch die Änderungen an #1649 wird auch die Query kürzer. Evtl reicht dies bereits aus, wir haben aber schon eine Verkürzung der Anzeige vorbereitet und werden diese bald auf ptah einspielen.

**Comment by thefass** [comment]
der Fehler ist noch komplizierter
das Kettensymbol führt auf die joint Titel > das ist richtig
das Buchsymbol führt auf die Autorensaeite > das ist falsch
das Autorensymbol fehlt ganz

**Comment by mtrojan-ub** [comment]
Behoben, kann aber voraussichtlich erst heute Mittag auf ptah eingespielt werden
